src/vellum/workflows/nodes/displayable/tool_calling_node/utils.py

```diff
@@ -69,20 +95,11 @@ def run(self) -> Iterator[BaseOutput]:
             yield output
 
 
-class DynamicSubworkflowDeploymentNode(SubworkflowDeploymentNode[ToolCallingState]):
+class DynamicSubworkflowDeploymentNode(FunctionCallNodeMixin, SubworkflowDeploymentNode[ToolCallingState]):
```

Comment:
[nit] Let's make these mixins the second class passed in. We _might_ have logic in our serialization layer that is doing stuff using `__bases__[0]` just in case